### PR TITLE
Feat/on suggest

### PR DESCRIPTION
**Pull request by TroutZen**
### Description

Adds hook for getting suggest list and activeSuggest when it changes. Useful if you want to render the list of suggests outside of react-geosuggest. Addresses issue raised (https://github.com/ubilabs/react-geosuggest/issues/338)

- [x] All tests passing
- [x] Created tests which fail without the change (if possible)
- [x] Extended the README / documentation, if necessary
- [x] Commits and PR follow conventions

### CODE REVIEW

**Comment by TroutZen** [comment]
@ro-ka - lemme know if you can ge

**Comment by ericorruption** [comment]
Any update on this one? Would love to use it since there's no way to manipulate the suggestion sorting yet.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## 2.6.0 (2017-09-12)
+
+
```
Comment (ro-ka) [inline CHANGELOG.md]: Please remove the version bump.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-geosuggest",
-  "version": "2.5.0",
+  "version": "2.6.0",
```
Comment (ro-ka) [inline package.json]: Please remove the version bump.

File: src/suggest-list.jsx [hunk 3]
```diff
@@ -55,7 +55,7 @@ export default class SuggestList extends React.Component {
     return <ul className={classes} style={this.props.style}>
       {this.props.suggests.map(suggest => {
         const isActive = this.props.activeSuggest &&
-            suggest.placeId === this.props.activeSuggest.placeId,
```
Comment (ro-ka) [inline src/suggest-list.jsx]: Please keep the formatting here.

File: src/prop-types.js [hunk 4]
```diff
@@ -1,7 +1,7 @@
 import PropTypes from 'prop-types';
 
 /**
- * Default values
+ * Prop Types
```
Comment (ro-ka) [inline src/prop-types.js]: 👍

**Comment by ro-ka** [comment]
Oh, and sorry for the long delay — I was on parental leave.  👶

File: src/suggest-list.jsx (hunk 3 above)
Comment (davidfurlong) [reply src/suggest-list.jsx]: What do you mean by this?

Comment (ro-ka) [reply src/suggest-list.jsx]: Two spaces got added with this PR.

**Comment by davidfurlong** [comment]
Please see https://github.com/ubilabs/react-geosuggest/pull/368 with the comments implemented and merge conflicts resolved. The built files aren't updated though, as the CONTRIBUTING doc wasn't clear on these, and generally having committed builds doesn't play nice with git

**Comment by ro-ka** [comment]
#368 fixed the issues and got merged, so I’m closing this.

**Pull request merged**